Python/ceval.c

```diff
@@ -4985,7 +4980,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             assert(oparg);
             oparg <<= 8;
             oparg |= _Py_OPARG(*next_instr);
-            NOTRACE_DISPATCH_SAME_OPARG();
```

Comment:
As written here, EXTENDED_ARG can't be followed by DO_TRACING, but EXTENDED_ARG_QUICK could. Shouldn't they have the same behavior with respect to tracing?

Reply:
This could probably be reverted to essentially the 3.10 code:


https://github.com/python/cpython/blob/e56116172ec1e6cab2da215034e2acc8433094db/Python/ceval.c#L4409-L4414